loadtest/store/memstore/random.go

```diff
@@ -139,7 +140,11 @@ func (s *MemStore) RandomUser() (model.User, error) {
 	s.lock.RLock()
 	defer s.lock.RUnlock()
 
-	if len(s.users) < 2 {
+	minLen := 1
```

Comment:
A note here would be useful to remind us why are we doing this, given that we stumbled on this twice.